site/content/docs/main/backup-reference.md

```diff
@@ -61,6 +61,19 @@ velero backup create --from-schedule example-schedule
 
 This command will immediately trigger a new backup based on your template for `example-schedule`. This will not affect the backup schedule, and another backup will trigger at the scheduled time.
 
+
+### Limitation
+Backups created from schedule can have owner reference to the schedule. This can be achieved by command:
+
+```
+velero schedule create --use-owner-references-in-backup <backup-name>
+```
+By this way, schedule is the owner of it created backups. This is useful for some GitOps scenarios, or the resource tree of k8s synchronized from other places.
+Please do notice there is also side effect that may not be expected. Because schedule is the owner, when the schedule is deleted, the related backups will be deleted by k8s controller, too. 
```

Comment:
I think it's deleted by garbage collection, not the controller?  I think we also wanna mention the side effect that the BSL sync controller will add the backup back again?

Reply:
GC is also controller in k8s. 
OK. I will add more detail in this document.

Reply:
yeah right I was thinking the schedule controller.  My bad.